test/appTest.js

```diff
@@ -0,0 +1,9 @@
+const assert = require('chai').assert;
+const index = require('../index');
+
+describe('test', () => {
+    it('Should return string', () => {
+        let result = 'ci with travis'
+        assert.equal(result,'ci with travis');
+    });
+});
```

Comment:
New Line.

Test file name should match with the file which you are testing. It should be `test/index.test.js`
Also I see the test cases are not matching with file or functions in `index.js` file which should be tested. But I know the purpose of this repo is not the write the functional code and test it , but to understand git pr life cycle and integration with travis. Hence this is okay for now.

Reply:
Changed the file name to index.test.js and added a new line.